includes/wc-payment-api/class-wc-payments-api-client.php

```diff
@@ -1556,11 +1557,34 @@ public function update_subscription_item( $wcpay_subscription_item_id, $data ) {
 	 * @throws API_Exception If payment method does not exist.
 	 */
 	public function get_payment_method( $payment_method_id ) {
-		return $this->request(
+		// A fallback for sites, which migrated from Stripe to WCPay.
+		$is_source = preg_match( '/^src_\w+$/i', $payment_method_id );
+		if ( $is_source ) {
+			$api = self::SOURCES_API;
+		} else {
+			$api = self::PAYMENT_METHODS_API;
+		}
```

Comment:
Ternary operator would be a better fit here IMO, it's short and clean.
```suggestion
		$api = $is_source ? self::SOURCES_API : self::PAYMENT_METHODS_API;
```

Reply:
Also done in https://github.com/Automattic/woocommerce-payments/pull/4663/commits/0ad203a85bf1c300d838519ab3eeec2d9e39dbd1.

FWIW my idea here was to use a full `if-else` block to attract attention, and focus on the point, but with the longer comment from the previous message, that's not needed already. Otherwise my thinking was kind of like this:

![image](https://user-images.githubusercontent.com/5311119/188086061-192372b4-3954-48ce-96e3-7eca58ceb165.png)

Reply:
Your assumption was fair, but it also requires some good understanding of old Stripe API (sources). Many new folks, including myself could not know these details, so more verbose comment should give enough hints on what's going on there :) Thanks for fixing.